lib/nve/nve-main-page.dart

```diff
@@ -4,23 +4,110 @@ class NveMainPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return new MaterialApp(
-      home: new FirstScreen(),
+      home: MyStatefulWidget(),
     );
   }
 }
 
-class FirstScreen extends StatelessWidget {
+class MyStatefulWidget extends StatefulWidget {
+  MyStatefulWidget({Key key}) : super(key: key);
+
+  @override
+  FirstScreen createState() => FirstScreen();
+}
+
+class FirstScreen extends State<MyStatefulWidget> {
+  int _selectedIndex;
+
+  @override
+  void initState() {
+    super.initState();
+
+    _selectedIndex = 0;
+  }
+
+  void _onItemTapped(int index) {
+    setState(() {
+    _selectedIndex = index;
+      switch (index) {
+        case 0:
+        {
+          Navigator.push(
+            context, 
+            new MaterialPageRoute(builder: (context) => new BluePage()),
+          );
+          break;
+        }
+        case 1:
+        {
+          Navigator.push(
+            context, 
+            new MaterialPageRoute(builder: (context) => new RedPage()),
+          );
+          break;
+        }
+        case 2:
+        {
+          Navigator.push(
```

Comment:
You should not open a new full screen page but a sub page located above the bottomNavigationBar